### DISCUSSION TITLE
Sigmoid function

### DISCUSSION THREAD

**Discussion by Knightrider16** [itsAnanth/xor-gate-perceptron#4] [Q&A] [answered]
Describe the sigmoid function given

**Comment by itsAnanth** [comment, marked as answer]
Sigmoid is the primary activation function used, we get the logits at the output layer, however we need to convert this to probabilities for this hence we use sigmoid. 

(1 / (1 + e^-x))
